### ISSUE TITLE
Testing Edit Profile Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4646]
# Page to Test:

Edit Profile - https://openlibrary.org/people/bpmcneilly?m=edit

# Steps

Steps for this task are outlined in #4521

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly Both images are added through background property
![image](https://user-images.githubusercontent.com/64412143/108588807-27c30600-7381-11eb-89a3-23f22dd115dc.png)

**Comment by bpmcneilly** [comment]
@Yashs911 - I think we may need to check in with @mekarpeles about this. 

Taking a quick look, there are definitely going to be a few issues we find with [WMD](https://github.com/internetarchive/wmd). But, I don't know if those are technically within scope of our Open Library review and/or if they should be flagged & managed directly within that repo.

**Comment by Yashs911** [comment]
@bpmcneilly Any update on this

**Comment by bpmcneilly** [comment]
Hi Yash, for these, we're going to flag these issues against the WMD repository, rather than openlibrary directly. We'll likely end up closing them out over there, but for now I think it makes sense to focus on testing & remediating issues that are directly caused by the openlibrary directly. I think we can close this if there's no other issues.
